go-controller/pkg/util/ovs.go

```diff
@@ -162,6 +162,18 @@ func run(cmdPath string, args ...string) (*bytes.Buffer, *bytes.Buffer, error) {
 	return stdout, stderr, err
 }
 
+func run(cmdPath string, args ...string) (*bytes.Buffer, *bytes.Buffer, error) {
+	return runCmd(runner.exec.Command(cmdPath, args...), cmdPath, args...)
+}
+
+func runWithEnvVars(cmdPath string, envVars []string, args ...string) (*bytes.Buffer, *bytes.Buffer, error) {
+	cmd := runner.exec.Command(cmdPath, args...)
+	if len(envVars) != 0 {
+		cmd.SetEnv(envVars)
```

Comment:
@girishmg we should actually always call SetEnv() to ensure consistent calls. If cmd.Env is nil then it uses the parent's environment, while here daemon-mode calls would use only the vars in envVars (which is only OVN_NB_DAEMON). Pretty big difference. Plus passing a minimal environment to nbctl is a good idea anyway.

Reply:
Good catch. Understood.